cms/middleware/utils.py

```diff
@@ -3,8 +3,10 @@
 from __future__ import absolute_import, unicode_literals
 
 from ..utils import apphook_reload
+from cms.utils.compat.dj import MiddlewareMixin
```

Comment:
I've actually done the other way around (correcting ..utils with cms.utils, which is the most used import style in the codebase)